core/modules/layout/layout.module

```diff
@@ -1221,18 +1221,18 @@ function layout_get_layout_by_path($path = NULL, $router_item = NULL) {
     // @ todo: This adds very specific code for node previews which too closely
     // couples Node and Layout modules. This needs to be reverted in a future
     // commit.
-    $data = NULL;
+    $preview_node = NULL;
     $node_types = node_type_get_types();
     foreach ($node_types as $type) {
-      $preview_path = 'node/' . $type->type . '/preview';
+      $preview_path = "node/preview/{$type->type}/%";
```

Comment:
We tend to opt for `$preview_path = "node/preview/' . $type->type . '/%"`

Reply:
Changed in the PR. (Guessing you intended to use single quotes all the way there.)

Reply:
Thanks, this is a micro-performance improvement we've been following throughout core because double quotes `"`" are slightly slower since PHP is parsing the string for those variable replacements. Single quotes `''` go a bit faster :)